tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -276,3 +308,13 @@ def _a11y_click_menu_item(self, menu_item_locator):
     def _wait_for_toggle_ready(self, by, locator):
         checkbox = self.marionette.find_element(by, locator)
         Wait(self.marionette).until(expected.element_enabled(checkbox))
+
+    # this method is a copy of go_back() method in regions/keyboard.py
+    def return_to_main_menu(self, gaia_header=True):
```

Comment:
Hmm, but I think not having gaia header in this case actually was a bug (I filed one). Technically it was a workaround to make it work with non-gaia header, so I don't think we should explicitly search for it, since it might delay the tests if we use the timeouts, and it will hide the fact that it does not use the gaia-header.